### ISSUE TITLE
✨ [Feature] Use contributors api to automatically generate contributors images in README

### ISSUE THREAD

**Issue by devhaozi** [goravel/goravel#394]
### Before feedback (在反馈之前)

- [X] There are no features that I want to submit in Issues (当前 Issues 中没有我将要提交的新功能)

### Describe Feature (描述功能)

Use the following code:

```
<a href="https://github.com/TheTNB/panel/graphs/contributors">
  <img alt="contributors" src="https://contrib.rocks/image?repo=goravel/framework"/>
</a>
```

Rendering:
<a href="https://github.com/TheTNB/panel/graphs/contributors">
  <img alt="contributors" src="https://contrib.rocks/image?repo=goravel/framework"/>
</a>

**Comment by hwbrzzl** [comment]
Good idea, one question, can it add users that aren't contributors to goravel/framework?

**Comment by devhaozi** [comment]
> Good idea, one question, can it add users that aren't contributors to goravel/framework?

Not support, I think that repositories such as goravel/gin can be configured with their own repository names.

**Comment by hwbrzzl** [comment]
A good way to generate contributors, but if can't custom the content, may be not suitable for us ~
